app/data/models/User.py

```diff
@@ -40,17 +48,18 @@ def validatePassword(self):
         # min 8 characters, max 20
         numberOfCharacters = r"\.{8, 20}"
 
-        return re.match(capitalLetter, password) and re.match(smallLetter, password) and re.match(oneNumber, password) and re.match(numberOfCharacters, password)
-    
-    def ifUserExists(self):
+        # return re.match(capitalLetter, password) and re.match(smallLetter, password) and re.match(oneNumber, password) and re.match(numberOfCharacters, password)
+
+        return True
 
-        if(UsersRepository.fetchUserByName(self.username)) or UsersRepository.fetchUserByEmail(self.email):
-            return True
+    def getUserDetails(self):
 
-        return False
+        return [self.id, self.username, self.email, self.password, self.birthday]
 
-    def saveUserToDataBase(self):
-        if self.ifUserExists():
-            return False
 
-        return UsersRepository.saveUser(self)
+    def ifUserExists(self):
+        return UsersRepository.fetchUserByEmail(self.email)
```

Comment:
fetchByEmail

Reply:
I see it has not changed, reply if you dont agree with remark.

Reply:
I read comments and then go to code and changed it. I want this method to use if a user already exists in the database. Maybe I should check if username or email is already in the database and then return true or false